types/selenium-webdriver/lib/webdriver.d.ts

```diff
@@ -1,5 +1,909 @@
+     */
+    newWindow(typeHint: string): Promise<void>;
+
+    /**
+     * Changes focus to the active modal dialog, such as those opened by
+     * `window.alert()`, `window.confirm()`, and `window.prompt()`. The returned
+     * promise will be rejected with a
+     * {@linkplain error.NoSuchAlertError} if there are no open alerts.
+     *
+     * @return {!AlertPromise} The open alert.
+     */
+    alert(): AlertPromise;
+}
+
+/**
+ * Each WebDriver instance provides automated control over a browser session.
+ *
+ */
+export class WebDriver{
+    /**
+     * @param {!(./session.Session|IThenable<!./session.Session>)} session Either
+     *     a known session or a promise that will be resolved to a session.
+     * @param {!command.Executor} executor The executor to use when sending
+     *     commands to the browser.
+     * @param {(function(this: void): ?)=} onQuit A function to call, if any,
+     *     when the session is terminated.
+     */
+    constructor(session: Session| Promise<Session>, executor: command.Executor, onQuit?: ((this: void) => any));
+
+    /**
+     * Creates a new WebDriver session.
+     *
+     * This function will always return a WebDriver instance. If there is an error
+     * creating the session, such as the aforementioned SessionNotCreatedError,
+     * the driver will have a rejected {@linkplain #getSession session} promise.
+     * This rejection will propagate through any subsequent commands scheduled
+     * on the returned WebDriver instance.
+     *
+     *     let required = Capabilities.firefox();
+     *     let driver = WebDriver.createSession(executor, {required});
+     *
+     *     // If the createSession operation failed, then this command will also
+     *     // also fail, propagating the creation failure.
+     *     driver.get('http://www.google.com').catch(e => console.log(e));
+     *
+     * @param {!command.Executor} executor The executor to create the new session
+     *     with.
+     * @param {!Capabilities} capabilities The desired capabilities for the new
+     *     session.
+     * @param {(function(this: void): ?)=} onQuit A callback to invoke when
+     *    the newly created session is terminated. This should be used to clean
+     *    up any resources associated with the session.
+     * @return {!WebDriver} The driver for the newly created session.
+     */
+    static createSession(...var_args: any[]): WebDriver;
+
+    /** @override */
+    execute(command:command.Command): Promise<void>;
+
+    /** @override */
+    setFileDetector(detector: FileDetector): void;
+
+    /** @override */
+    getExecutor(): command.Executor;
+
+    /** @override */
+    getSession(): Promise<Session>;
+
+    /** @override */
+    getCapabilities(): Promise<Capabilities>;
+
+    /** @override */
+    quit(): Promise<void>;
+
+    /** @override */
+    actions(options: {async: (boolean|undefined), bridge: (boolean|undefined)}): Actions;    
```

Comment:
Hi, this is causing typescript error.
```
TS2345: Argument of type '{ bridge: true; }' is not assignable to parameter of type '{ async: boolean; bridge: boolean; }'.
  Property 'async' is missing in type '{ bridge: true; }' but required in type '{ async: boolean; bridge: boolean; }'
```
I have create a PR for fix.
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69750